invisible_cities/cities/detsim_simulate_electrons.py

```diff
@@ -0,0 +1,152 @@
+import numpy as np
+
+from typing import List
+from typing import Tuple
+
+# from invisible_cities.cities.detsim_get_psf import binedges_from_bincenters
+
+#######################################
+######### ELECTRON SIMULATION #########
+#######################################
+def generate_ionization_electrons(energies    : np.ndarray,
+                                  wi          : float = 22.,
+                                  fano_factor : float = 0.1) -> np.ndarray:
```

Comment:
```suggestion
def generate_ionization_electrons(energies    : np.ndarray,
                                  wi          : float = 22 * units.eV,
                                  fano_factor : float = 0.1) -> np.ndarray:
```
Are these values correct? I think 22.4 and 0.15 are the correct ones.